src/main/kotlin/no/nav/hjelpemidler/delbestilling/hjelpemidler/Hjelpemidler.kt

```diff
@@ -17,6 +17,11 @@ object HjelpemiddelDeler {
 
         return HjelpemiddelMedDeler(hjelpemiddel.navn, hjelpemiddel.hmsnr, deler)
     }
+
+    fun hentAlleHjelpemidlerMedDeler(): List<HjelpemiddelMedDeler> {
+        return hjelpemidler.map { hentHjelpemiddelMedDeler(it.hmsnr) }.filterNotNull()
```

Comment:
Det er en fin tanke å gjenbruke `hentHjelpemiddelMedDeler` her, men det er et par utfordringer med det:
1. Det er først en loop for å mappe over `hjelpemidler`, deretter inni `hentHjelpemiddelMedDeler` er det en `find` som gjøres for å finne riktig hjelpemiddel. Det blir kanskje litt dobbelt opp?
2. Inni `hentHjelpemiddelMedDeler` logger vi for å se hva som har blitt funnet. Disse kan nok skape litt støy hvis den funksjonen også kalles for hvert hjelpemiddel via `hentAlleHjelpemidlerMedDeler`.

Foreslår derfor å forenkle det, ala:
```suggestion
        return hjelpemidler.map { HjelpemiddelMedDeler(it.navn, it.hmsnr, delerPerHjelpemiddel[it.type]) }
```